packages/web-components/src/components/locale-modal/locale-modal.scss

```diff
@@ -18,6 +18,10 @@
   #{$dds-prefix}-expressive-modal-header {
     padding: 0 20% 0 $spacing-05;
     margin-bottom: $carbon--spacing-03;
+
+    > * {
+      width: fit-content;
+    }
```

Comment:
Could you explain what this style rule is for?

Reply:
When using `LinkWithIcon`, the component takes the whole width of the Locale Modal Header, so when tabbing into focus there's a long rectangle focusing on empty space. Clicking on the empty space is also considered as if the content is being clicked. Using this style rule makes it so it only wraps around the actual content and avoid these problems.

Btw, I tried using `display: inline-block`. While it also works, it shifts the component down an unspecified amount of pixels and I wasn't able to replicate the original positioning.

Reply:
I see, thanks @IgnacioBecerra for explaining! Looks that you wanted `:host(#{$dds-prefix}-locale-modal) #{$dds-prefix}-expressive-modal-heading { width: fit-content }`? We should try to avoid `*` selector unless it's absolutely necessary to make it more specific and avoid selecting elements that we don't intend.

Reply:
Oh I see, that's what I wanted, thanks! 

Before using the `*` selector, I tried many selector combinations (including a similar one to the one above, except I used `:host` instead for the second one). The nuances about Web Component styling vs regular are interesting, thanks for always helping out with them @asudoh !